src/frontend/js/contexts/SessionContext/JoanieSessionProvider.tsx

```diff
@@ -119,11 +122,20 @@ const JoanieSessionProvider = ({ children }: React.PropsWithChildren<{}>) => {
     () => ({
       user,
       isPending,
+      joanieUserProfile: joanieUserProfile.item,
       destroy,
       login,
       register,
     }),
-    [user, isPending, destroy, login, register],
+    [
+      user,
+      isPending,
+      joanieUserProfile.item,
+      joanieUserProfile.states.fetching,
```

Comment:
```suggestion
```

Reply:
I'll totally remove profile information from this context. In this way there is only one way to retrieve user profile information, the hook `useJoanieUserProfile`